### PR TITLE
add AWS support for DeleteIntermediateFiles task

### PR DESCRIPTION
**Pull request by henriqueribeiro**
Add support for AWS in `DeleteIntermediateFiles` task in `Module00a`.

### CODE REVIEW

File: wdl/Module00a.wdl [hunk 1]
```diff
@@ -356,6 +358,7 @@ task DeleteIntermediateFiles {
   input {
     Array[File] intermediates
     Array[File]? dummy # Pass in outputs that must be complete before cleanup
+    String cloud_sdk_docker
```
Comment (mwalker174) [inline wdl/Module00a.wdl:361]: Can you add a comment here suggesting which images to use for GCS and AWS?

**Review by mwalker174 — CHANGES_REQUESTED** [review]
Thanks for this @henriqueribeiro ! We do need to have the input templates in `/test_input_templates` updated as well (`batch`, `module00a`, and `single-sample` I believe). You can validate that all generated inputs are valid using `scripts/test/validate.sh`.

**Comment by henriqueribeiro** [comment]
I believe we need to update the `scripts/test/validate.sh`. It seems outdated and not working correctly. Do you confirm it needs some work?

**Comment by mwalker174** [comment]
@henriqueribeiro It appears the script is working correctly, but there are 2 errors in the WDLs that cause it to fail immediately. I've created PR #118 to fix that.

**Comment by henriqueribeiro** [comment]
@mwalker174 all tests pass now :)

**Review by mwalker174 — APPROVED** [review]
Great, looks good to me!

**Pull request merged**